utils/host_utils.cpp

```diff
@@ -221,10 +228,10 @@ void constructWilsonTestSpinorParam(quda::ColorSpinorParam *cs_param, const Quda
     cs_param->nDim = 4;
   }
   for (int d = 0; d < 4; d++) cs_param->x[d] = gauge_param->X[d];
-  bool pc = (inv_param->solution_type == QUDA_MATPC_SOLUTION || inv_param->solution_type == QUDA_MATPCDAG_MATPC_SOLUTION);
+  bool pc = isPCSolution(inv_param->solution_type);
```

Comment:
This also needs to be propagated into `utils/staggered_host_utils.cpp`

Reply:
7052e04